Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -43,17 +43,19 @@ void BatteryStateOfChargeService::addData(const BatteryData& batteryData)
     // This is where you can update your variables
     // Hint: There are many different ways that the totalAmpHoursUsed can be updated
     // i.e: Taking a running average of your data values, using most recent data points, etc.
+    const double ms_hour_conversion = 3600000.0;
```

Comment:
Common convention is to declare magic constant numbers in the namespace section at the top of the file. This way, all functions within the file have access to the constant if needed.
Make sure you use all uppercase for the constant numbers.